accel/kvm/kvm-all.c

```diff
@@ -2171,11 +2171,12 @@ static void kvm_handle_io(uint16_t port, MemTxAttrs attrs, void *data, int direc
 {
```

Comment:
As discussed, let's drop this commit for now. We can fix debug access in a separate submission